jmxtrans-output/jmxtrans-output-core/src/test/java/com/googlecode/jmxtrans/model/output/support/GraphiteWriterFactoryTest.java

```diff
@@ -178,6 +180,83 @@ public void writer_using_udp_protocol() throws LifecycleException, URISyntaxExce
 		}
 	}
 
+	@Test
+	public void socketExpirationIsUsedByDefault() throws LifecycleException, URISyntaxException {
+		ImmutableList<Server> servers = configurationParser.parseServers(ImmutableList.of(file("/graphite-writer-factory-example2.json")), false);
+
+		assertThat(servers).hasSize(1);
```

Comment:
There are a lot of assertions here that don't really have anything to do with the name of the test. They should probably be removed.

Reply:
Removed these.